CMakeLists.txt

```
@@ -1776,6 +1778,9 @@ elseif(UNIX AND NOT APPLE AND NOT RISCOS AND NOT HAIKU)
     # Need to check for ROCKCHIP platform and get rid of "Can't window GBM/EGL surfaces on window creation."
     CheckROCKCHIP()
     CheckX11()
+    if(PKG_CONFIG_FOUND)
+      CheckFribidi()
+    endif()
```

There are already pkg-config checks inside the macro.
```suggestion
    CheckFribidi()
```

---

Done.